diff --git a/vhdl_pooyan_rev_0_2_2020_04_26/rtl_t80_350/T80.vhd b/vhdl_pooyan_rev_0_2_2020_04_26/rtl_t80_350/T80.vhd
--- a/vhdl_pooyan_rev_0_2_2020_04_26/rtl_t80_350/T80.vhd
+++ b/vhdl_pooyan_rev_0_2_2020_04_26/rtl_t80_350/T80.vhd
@@ -734,7 +734,7 @@
 						F(Flag_N) <= DI_Reg(7);
 						F(Flag_C) <= ioq(8);
 						F(Flag_H) <= ioq(8);
-						ioq := (ioq and x"7") xor ('0'&BusA);
+						ioq := (ioq and '0'&x"07") xor ('0'&BusA); -- RB: modified for Vivado error "operands of logical operator '&' have different lengths ... :562"
 						F(Flag_P) <= not (ioq(0) xor ioq(1) xor ioq(2) xor ioq(3) xor ioq(4) xor ioq(5) xor ioq(6) xor ioq(7));
 					end if;
 
